stereo calibration/CalibrationDataCollection.md

```diff
@@ -1,65 +1,105 @@
+
+All of these become challenging with a wide FOV (> 1m width)
 
 ## Recommendations
 
 ### 0. Arrangement of the cameras
-There are very nice recommendations on the arrangement of the cameras in the [StereoMorph docs](https://aaronolsen.github.io/tutorials/stereomorph/arranging_cameras_photography.html)
-- As much FOV overlap as possible? --- see StereoMorph
-- Remember! We are calibrating the cameras for a specific relative position of the cameras, zoom and focus levels!
-    - From the [choosing cameras section](https://aaronolsen.github.io/tutorials/stereomorph/choosing_cameras_general.html) > "the resulting calibration is specific to that particular arrangement of the cameras (their position and orientation). The cameras can move if they are rigidly attached to some support system and the support system itself is moved. However, the relative position and orientation of the cameras relative to one another must remain fixed to collect accurate 3D data. Additionally, the calibration is specific to the particular zoom and focus, so these must also remain fixed.
-    - 
-- Set the cameras to the smallest aperture
+There are very nice recommendations on the arrangement of the cameras in the [StereoMorph docs](https://aaronolsen.github.io/tutorials/stereomorph/arranging_cameras_photography.html) (although they focus on taking pictures to reconstruct a 3D object)
+- Remember! We are calibrating the cameras for a specific relative position of the cameras, but also zoom (focal length) and focus levels!
+    - All must stay constant during calibration and data collection (see the [choosing cameras section](https://aaronolsen.github.io/tutorials/stereomorph/choosing_cameras_general.html)
+- Re camera settings
+    - They recommend to turn off auto-focus and auto-zoom modes, and vibration reduction (VR) if the lens has it.
+    - They recommend setting the cameras to the smallest aperture
+        > "The smaller the aperture, the greater the depth of field (i.e. the more things are in focus both close and far away from the camera). This is essential in a stereo camera setup because in order to digitize points accurately throughout the calibration volume they must be in focus."
+
+- Re cameras' positions:
+    - "Because the camera is often positioned half a meter or more away from the object, even a small shift of the camera can translate into a large shift in the image frame, causing large inaccuracies"
+    - Leg stoppers ([this type](https://www.amazon.co.uk/Universal-Stainless-Photography-Accessories-Replacement/dp/B0B9NXFTWP/ref=sr_1_11?keywords=tripod+rubber+feet+replacement&qid=1689006351&sr=8-11) or [these with spikes for uneven surfaces](https://www.amazon.co.uk/Universal-Non-Slip-Replacement-Compatible-Kingjoy/dp/B0B33RDMTC/ref=sr_1_12?keywords=tripod+rubber+feet+replacement&qid=1689006351&sr=8-12)) could be useful to keep the tripods as fixed as possible against the wind and on top of uneven terrain
```

Comment:
Great idea! Our tripods aren't compatable with this, but we're getting wind shields (which will hopefully do the trick)